tests/postgres_tests/test_hstore.py

```diff
@@ -20,6 +21,32 @@ class HStoreTestCase(PostgreSQLTestCase):
     pass
 
 
+class HstoreOIDTests(HStoreTestCase):
+
+    def test_hstore_at_most_zero_queries(self):
+        from django.contrib.postgres.signals import get_hstore_oids
```

Comment:
Are the inner imports needed? Can they go in the try/except block at the top of the file?